README.md

```diff
@@ -1,7 +1,7 @@
 # GitHub Teacher's Blog
 
 ### What You'll Find Here
-This is where GitHubTeacher puts thoughts and experiments using GitHub Pages and Jekyll. Please feel free to open any pull requests or ask any questions about anything you see here! 
+This is where GitHubTeacher puts thoughts and experiments using GitHub Pages and Jekyll. Please feel free to open any pull requests or ask any questions about anything you see here! :tada: :sparkles: :pizza:
```

Comment:
@githubteacher I think this looks great!

Reply:
Umm, pizza.